contracts/ROSCA.sol

```diff
@@ -39,15 +39,16 @@ contract ROSCA {
   // TODO(ron): replace this with an actual wallet. Right now this is accounts[9] of the testrpc used
   // by tests.
   // WeTrust's account from which Escape Hatch can be enabled.
-  address constant internal ESCAPE_HATCH_ENABLER = 0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e;
+  address constant internal ESCAPE_HATCH_ENABLER = 0x7f299eb63832d033a1252e8138574a202E47B92e;
 
   /////////
   // EVENTS
   /////////
   event LogContributionMade(address user, uint256 amount, uint256 currentRound);
   event LogStartOfRound(uint256 currentRound);
+  event LogBidSurpassed(uint256 prevBid, address prevWinnerAddress, uint256 currentRound);
   event LogNewLowestBid(uint256 bid, address winnerAddress, uint256 currentRound);
-  event LogRoundFundsReleased(address winnerAddress, uint256 amount, uint256 currentRound);
+  event LogRoundFundsReleased(address winnerAddress, uint256 amount, uint256 currentRound, uint256 roundDiscount);
```

Comment:
just to be consistent that currentRound is always the last parameter

Reply:
i wanted it to be consistent for currentRound to be 3rd

Reply:
i've changed it